app/controllers/posts_controller.rb

```diff
@@ -25,6 +25,19 @@ def create
     end
   end
 
+  # def destroy
+  #   @post = Post.find(params[:post_id])
+  #   @post.destroy
+  # end
```

Comment:
- [ ] Please remove any unused code for a cleaner and more optimized code
- [ ] Also, as per project requirements, please consider using the CanCanCan gem built-in method `load_and_authorize_resource` inside your posts controller to invoke authorization for users while creating and/or deleting posts. 🙏